core/experiments/manager.php

```diff
@@ -441,7 +441,7 @@ private function add_default_features() {
 			'tag' => esc_html__( 'Performance', 'elementor' ),
 			'description' => esc_html__( 'Reduce the DOM size by eliminating HTML tags in various elements and widgets. This experiment includes markup changes so it might require updating custom CSS/JS code and cause compatibility issues with third party plugins.', 'elementor' ),
 			'release_status' => self::RELEASE_STATUS_ALPHA,
-			'default' => self::STATE_INACTIVE,
+			'default' => self::STATE_ACTIVE,
```

Comment:
```suggestion
			'default' => self::STATE_INACTIVE,
			'new_site' => [
				'default_active' => true,
				'minimum_installation_version' => '3.26.0',
			],
```

Reply:
@rami-elementor This is not real, I just did this to see what breaks :) But I will update it if this is the requirement, thanks!